DEVICE_ID: demo
Body:
  base_link: base_link
  frame: sensor
  extrinsic:
    translation:
      x: 0.0
      y: 0.0
      z: 0.0
    quaternion:
      x: 0.0
      y: 0.0
      z: 0.0
      w: 1.0
Sensor:
  Lidar:
    base_link: sensor
    frame: lidar
    extrinsic:
      translation:
        x: 0.0
        y: 0.030219
        z: -0.02354
      quaternion:
        x: 0.0
        y: -0.0
        z: 0.0
        w: 1.0
  Camera:
    base_link: sensor
    frame: camera
    extrinsic:
      translation:
        x: -0.01
        y: 0.02
        z: -0.04708
      quaternion:
        x: 0.5000000001025516
        y: -0.49999999979489657
        z: 0.5000000001025517
        w: -0.5
    # for camera2lidar calibration demo data
    intrinsic:
      model: Pinhole
      int_matrix:
      - 799.4381517411
      - 0
      - 956.341097
      - 0
      - 798.6048946773
      - 571.855315
      - 0
      - 0
      - 1
      dist_coeff:
      - 0.4631934253
      - 0.0425895801
      - -9.26186e-05
      - 4.64662e-05
      - 0.001872107
      - 0.811456
      - 0.122953
      - 0.008943
      image_size:
      - 1920
      - 1080
    # for camera2imu calibration demo data
    # intrinsic:
    #   model: Pinhole
    #   int_matrix:
    #   - 792.4640737604
    #   - 0
    #   - 965.645594
    #   - 0
    #   - 791.1053752151
    #   - 547.250649
    #   - 0
    #   - 0
    #   - 1
    #   dist_coeff:
    #   - -0.0352108256
    #   - 0.147926732
    #   - 1.2937e-05
    #   - 0.0001654014
    #   - 0.014088817
    #   - 0.290436
    #   - 0.095993
    #   - 0.061735
    #   image_size:
    #   - 1920
    #   - 1080
  IMU:
    base_link: sensor
    frame: imu
    extrinsic:
      translation:
        x: 0.0
        y: 0.0
        z: 0.0
      quaternion:
        x: 0.0
        y: 0.0
        z: 0.0
        w: 1.0
